crates/bevy_app/src/main_schedule.rs

```diff
@@ -142,13 +142,41 @@ impl Main {
         }
 
         world.resource_scope(|world, order: Mut<MainScheduleOrder>| {
+            let step = world
+                .get_resource_mut::<Stepping>()
+                .and_then(|mut s| s.try_step());
             for label in &order.labels {
+                if let Some(false) = step {
+                    let label = &**label;
+                    if label.eq(&Update) || label.eq(&RunFixedUpdateLoop) {
+                        continue;
+                    }
+                }
                 let _ = world.try_run_schedule_ref(&**label);
             }
         });
     }
 }
 
+#[derive(Resource)]
+pub enum Stepping {
+    Disabled,
+    Enabled { frames: usize },
```

Comment:
I'm not sold on the value of the `frames` field here: maybe the docs will clarify it further.

Reply:
Yeah im not convinced either. Just a convenience at this point / provides functionality to help test things.